src/main/scala/Compiler.scala

```diff
@@ -4,6 +4,7 @@ import java.io.{File, FileWriter, Writer}
 
 import essent.Emitter._
 import essent.Extract._
+import essent.EssentVcd
```

Comment:
When compiling from scratch, this produces a warning "imported `EssentVcd` is permanently hidden by definition of object EssentVcd in package essent"

Reply:
Resolved this with recent commit